base_location_geonames_import/tests/test_base_location_geonames_import.py

```diff
@@ -166,3 +176,93 @@ def test_import_duplicated_city_name(self):
                 state,
                 "Incorrect state for {} {}".format(state_name, zip_code),
             )
+
+    def test_import_countries(self):
+        max_import = 10
+        self.wizard_2.with_context(max_import=max_import).run_import()
+        # Look if there are imported states for the country
+        state_country_2_count = self.env["res.country.state"].search_count(
+            [("country_id", "=", self.country_2.id)]
+        )
+        state_country_3_count = self.env["res.country.state"].search_count(
+            [("country_id", "=", self.country_3.id)]
+        )
+        self.assertTrue(state_country_2_count)
+        self.assertTrue(state_country_3_count)
+        # Look if there are imported zips
+        zip_country_2_count = self.env["res.city.zip"].search_count(
+            [("city_id.country_id", "=", self.country_2.id)]
+        )
+        zip_country_3_count = self.env["res.city.zip"].search_count(
+            [("city_id.country_id", "=", self.country_3.id)]
+        )
+        self.assertEqual(zip_country_2_count, max_import)
+        self.assertEqual(zip_country_3_count, max_import)
+
+        # Look if there are imported cities
+        city_country_2_count = self.env["res.city"].search_count(
+            [("country_id", "=", self.country_2.id)]
+        )
+        city_country_3_count = self.env["res.city"].search_count(
+            [("country_id", "=", self.country_3.id)]
+        )
+        self.assertTrue(city_country_2_count)
+        self.assertTrue(city_country_3_count)
+
+        # Reimport again to see that there's no duplicates
+        self.wizard.with_context(max_import=max_import).run_import()
+        state_2_count2 = self.env["res.country.state"].search_count(
+            [("country_id", "=", self.country_2.id)]
+        )
+        state_3_count2 = self.env["res.country.state"].search_count(
+            [("country_id", "=", self.country_3.id)]
+        )
+        self.assertEqual(state_country_2_count, state_2_count2)
+        self.assertEqual(state_country_3_count, state_3_count2)
+
+        city_2_count2 = self.env["res.city"].search_count(
+            [("country_id", "=", self.country_2.id)]
+        )
+        city_3_count2 = self.env["res.city"].search_count(
+            [("country_id", "=", self.country_3.id)]
+        )
+        self.assertEqual(city_country_2_count, city_2_count2)
+        self.assertEqual(city_country_3_count, city_3_count2)
+
+        zip_count = self.env["res.city.zip"].search_count(
+            [("city_id.country_id", "in", (self.country_2.id, self.country_3.id))]
+        )
+        self.assertEqual(zip_count, 20)
+
+    def test_import_country_single_value(self):
+        max_import = 5
```

Comment:
No need to add this test, and more with the same invalid asserts than the previous. For checking this backward compatibility, just don't change in https://github.com/OCA/partner-contact/pull/884/files#diff-1718593fd8e26e4fec3112ad5a4657c4R120 the assigned value and everything should work the same.